### PR TITLE
Added email for users with null workTime

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

File: src/TimeService.Business/Helpers/Emails/EmailSender.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    private readonly IEmailService _emailService;
+    private readonly ITextTemplateService _textTemplateService;
+    private readonly ITextTemplateParser _parser;
+
+    private async Task NotifyAsync(
+      UserData user,
+      string locale)
+    {
+      IGetTextTemplateResponse textTemplate = await _textTemplateService
+        .GetAsync(TemplateType.EmptyUserWorktimes, locale, null);
```
Comment (MariyaSin) [inline src/TimeService.Business/Helpers/Emails/EmailSender.cs:28]: remove null, you have default in the method

File: src/TimeService.Broker/Requests/EmailService.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      _rcSendEmail = rcSendEmail;
+      _logger = logger;
+    }
+
+    public async Task SendAsync(string email, string subject, string text, List<string> errors = null)
+    {
+      if (!await RequestHandler.ProcessRequest<ISendEmailRequest, bool>(
+        _rcSendEmail,
+        ISendEmailRequest.CreateObj(
+          email,
```
Comment (MariyaSin) [inline src/TimeService.Broker/Requests/EmailService.cs:29]: email:

File: src/TimeService.Broker/Requests/EmailService.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          subject,
```
Comment (MariyaSin) [inline src/TimeService.Broker/Requests/EmailService.cs:30]: subject: subject

File: src/TimeService.Broker/Requests/EmailService.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          text,
```
Comment (MariyaSin) [inline src/TimeService.Broker/Requests/EmailService.cs:31]: text: text

**Review by MariyaSin — APPROVED** [review]

File: src/TimeService.Business/Helpers/Emails/EmailSender.cs (hunk 1 above)
Comment (VLiamin) [reply src/TimeService.Business/Helpers/Emails/EmailSender.cs]: done

File: src/TimeService.Broker/Requests/EmailService.cs (hunk 2 above)
Comment (VLiamin) [reply src/TimeService.Broker/Requests/EmailService.cs]: done

File: src/TimeService.Broker/Requests/EmailService.cs (hunk 4 above)
Comment (VLiamin) [reply src/TimeService.Broker/Requests/EmailService.cs]: done

File: src/TimeService.Broker/Requests/EmailService.cs (hunk 3 above)
Comment (VLiamin) [reply src/TimeService.Broker/Requests/EmailService.cs]: done

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**